main.py

```diff
@@ -0,0 +1,55 @@
+import os
+import openai
+
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+
+from app import crud
+from app.session import get_db
+from app.schemas import UserCreate, ConversationCreate, MessageBase, UserUpdate, ConversationResponse, UserResponse
+
+openai.api_key = os.environ.get("OPENAI_API_KEY")
+
+app = FastAPI()
+
+
+@app.post("/users", status_code=201, response_model=UserCreate)
+def create_user(user: UserCreate, db: Session = Depends(get_db)):
+    db_user = crud.create_user(db, user.user_id)
+    return db_user
+
+
+@app.post("/chat")
+def get_chatgpt_response(message: MessageBase) -> str:
+    messages = [{"role": "user", "content": message.prompt}]
+    response = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo",
```

Comment:
Save that as a const at the beginning of the file.
ie.
`MODEL="gpt-3.5-turbo"`